docs/checkit.js

```diff
@@ -817,10 +770,26 @@ var Calendar = function(state, checkitObj) {
     //user wants to track
     self.numberOfMonths = self.endDate.diff(self.startDate, 'months', true);
     self.monthObjects = self.generateMonthObjects();
-    self.checkit = checkitObj;
-    
+    // Last month to be added, this is so that when you need to attach the
+    // click handler to this month, you don't do it to the entire calendar.
+    self.lastMonth = null;
+
 }
 
+Calendar.prototype.addMonth = function() {
+    // Add a month to the calendar.
+    var self = this;
+    
+    // Update self.endDate so that it occurs one month later.
+    self.endDate = self.endDate.add(1, 'months');
+    // Update the self.lastMonth property
+    self.lastMonth = new Month(self.endDate.format("YYYYMMDD"));
+    // Update the endDateString in the calendarState
+    self.state.endDateString = self.endDate.format("YYYYMMDD");
+    // Update the self.monthObjects so that it includes the new month
+    self.monthObjects = self.generateMonthObjects();
+};
+
     
 Calendar.prototype.generateMonthObjects = function() {
```

Comment:
If this function took `startDate` and `endDate` as parameters you could also use it to create a single month as needed.